extensions/data-workspace/.vscodeignore

```diff
@@ -0,0 +1,6 @@
+coverageConfig.json
+coverage
+out
+src
+tsconfig.json
```

Comment:
Why??

Reply:
Actually most of this file, why?

Reply:
the vscodeignore file is used for vsce package command, decides which files will be included in the vsix file. I've updated it and removed out from this file.

Reply:
Hmm don’t those files also impact the search or am I mistaken?

Reply:
@aaomidi No - that's a separate VS Code setting https://github.com/microsoft/azuredatastudio/blob/main/.vscode/settings.json#L16